pkg/asset/installconfig/powervs/session.go

```diff
@@ -21,8 +22,13 @@ var (
 // Session is an object representing a session for the IBM Power VS API.
 type Session struct {
 	Session *ibmpisession.IBMPISession
+	Creds   *UserCredentials
 }
 
+type UserCredentials struct {
+        ApiKey string
```

Comment:
I think the linter will complain about this not being APIKey

Reply:
Kinda silly but alright